packages/beacon-node/test/e2e/network/gossipsub.test.ts

```diff
@@ -138,6 +138,70 @@ function runTests({useWorker}: {useWorker: boolean}): void {
     );
   });
 
+  it("Publish and receive an attesterSlashing", async function () {
```

Comment:
The amount of copy pasted code is quite high in this file, maybe something that could be improved in this PR (or later one) to make it easier to add new events in the future

Reply:
I would suggest to do that in a different PR for extra clarity.